tests/Kernel/CorporateBlocks/CorporateBlocksSiteSwitcherTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme\Kernel\CorporateBlocks;
+
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Test site switcher block rendering.
+ */
+class CorporateBlocksSiteSwitcherTest extends CorporateBlocksTestBase {
+
+  /**
+   * Array containing data to merge with config object.
+   */
+  protected function getTestConfigData(): array {
+    return [
+      'info_label' => 'Info Lable',
+      'info_href' => 'https://info.com/domain',
+      'political_label' => 'Political label',
+      'political_href' => 'https://digit.com/domain',
+      'active' => 'info',
+    ];
+  }
+
+  /**
+   * Test site switcher block rendering.
+   */
+  public function testSiteSwitcherBlockRendering(): void {
```

Comment:
Can we move the public methods (the API) to the top and the helper methods below. Also in `CorporateBlocksFooterTest`?

Reply:
Fixed